components/movementsensor/gpsrtk/model_test.go

```diff
@@ -32,7 +32,7 @@ func TestGPSModels(t *testing.T) {
 		}`
 		_, err := config.FromReader(ctx, "", strings.NewReader(cfg1), logger)
 		test.That(t, err, test.ShouldNotBeNil)
-		test.That(t, err.Error(), test.ShouldContainSubstring, gpsrtk.ErrRoverValidation.Error())
+		test.That(t, err.Error(), test.ShouldContainSubstring, gpsrtk.ErrCorrectionSourceValidation.Error())
```

Comment:
if this is the only reason to make that error public, we can do away with ti, this particular test needs to be rewritten.

Reply:
yeah this is the only reason, will remove this test and make it public